closed/autoconf/custom-hook.m4

```diff
@@ -187,8 +187,11 @@ AC_DEFUN_ONCE([OPENJ9_THIRD_PARTY_REQUIREMENTS],
     AC_MSG_ERROR([Cannot continue])
   else
     AC_MSG_RESULT([yes])
-    AC_CHECK_FILE($with_freemarker_jar,, AC_MSG_ERROR([freemarker.jar not found at $with_freemarker_jar]))
+    AC_MSG_CHECKING([checking that $with_freemarker_jar exists])
+  if ! test -f $with_freemarker_jar; then
+    AC_MSG_ERROR([freemarker.jar not found at $with_freemarker_jar])
```

Comment:
You should use double-quotes to avoid problems when `$with_freemarker_jar` is empty or contains spaces.
Please also indent this code to match its surroundings.